src/components/measurements/measurementsD3.js

```diff
@@ -211,8 +212,26 @@ export const drawMeasurementsSVG = (ref, svgData) => {
       .call(
         axisLeft(yScale)
           .tickValues([yScale((layout.yMax - layout.yMin) / 2)])
-          .tickFormat(() => groupingValue))
-      .call((g) => g.attr("font-family", null));
+          .tickSize(layout.yAxisTickSize)
+          .tickFormat(groupingValue))
+      .call((g) => {
+        g.attr("font-family", null);
+        // If necessary, scale down the text to fit in the available space for the y-Axis labels
+        // This does mean that if the text is extremely long, it can be unreadable.
+        // We can improve on this by manually splitting the text into parts that can fit on multiple lines,
+        // but there's always limits of the available space so punting that for now.
```

Comment:
I think it's worth spending 30 minutes to see if an acceptable solution can be found here. There's _heaps_ of vertical pixel space [1]. Splitting on spaces, hypens, slashes etc should allow a reasonably good result.

[1] at least, there is in the measurements paper's h3n2 dataset I'm using to test. Even when colouring by a unchanging genotype, so there can only ever be one category. Perhaps in some configurations this is different?

Reply:
Yeah, I didn't want to go too far down the SVG text rabbit hole here...From what I understand, we have to draw the text to get the actual rendered width. So we split the text by spaces/hyphens/slashes, then render each substring to figure out the best configuration to fit the substrings into the allotted width. I can play around with this for a little bit.

Would we want to split reference strain names into multiple lines or would we have to special case them to preserve them in one line?